mavproxy/source/docs/modules/movinghome.rst

```diff
@@ -0,0 +1,25 @@
+.. _movinghome:
+
+==========
+Movinghome
+==========
+
+Allows changing vehicle's home position to new GCS location, if the GCS position moves. Requires NMEA GPS attached to serial port of GCS.
+
+.. code:: bash
+
+    module load movinghome
+
+Usage
+=====
+
+movinghome <status|on|off|radius|device|baud>
+
+- on: enable position updates as GCS moves. If GSC moves beyond thresold, vehicle home will be updated to current GCS position.
```

Comment:
typo in "threshold" (I know, any typos are most likely copied from my document)

Reply:
fixed